lib/ret_web/controllers/api-internal/v1/change_login_email_controller.ex

```diff
@@ -0,0 +1,37 @@
+defmodule RetWeb.ApiInternal.V1.ChangeLoginEmailController do
+  use RetWeb, :controller
+
+  alias Ret.Login
+
+  def post(conn, %{"old_email" => old_email, "new_email" => new_email})
+  when is_binary(old_email) and is_binary(new_email) do
+    conn = put_resp_header(conn, "content-type", "application/json")
+
+    with false <- is_empty_or_whitespace(old_email),
+         false <- is_empty_or_whitespace(new_email),
```

Comment:
If it’s valid, then it won’t be empty or whitespace. These `is_empty_or_whitespace/1` calls can be removed.